src/Vezeeta.Domain/Entities/Doctor.cs

```diff
@@ -1,8 +1,25 @@
-﻿using Vezeeta.Domain.Common;
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
 
-namespace Vezeeta.Domain.Entities;
-
-public sealed class Doctor : BaseEntity<string>
+namespace Vezeeta.Domain.Entities
 {
-    // TODO: Fill with properties
-}
\ No newline at end of file
+    public class Doctor
```

Comment:
For all entities, you should be inheriting from `BaseEntity<T>` where T is the datatype of the ID for your entity.